.devcontainer/Dockerfile

```diff
@@ -0,0 +1,13 @@
+# This dockerfile is used to build the devcontainer environment.
+# more info about vscode devcontainer: https://code.visualstudio.com/docs/devcontainers/containers
+FROM ubuntu
+ARG PYTHON_VERSION=3.9
+
+ENV DEBIAN_FRONTEND=noninteractive
+RUN apt-get update && apt install -yq  software-properties-common
+RUN add-apt-repository ppa:deadsnakes/ppa
+RUN apt-get install -yq python${PYTHON_VERSION} python3-pip python${PYTHON_VERSION}-dev python3-venv python${PYTHON_VERSION}-distutils python3-psutil
+RUN apt-get install -yq git build-essential liblz4-dev libunwind-dev
+
+RUN pip install pipx
+ENV PATH=/root/.local/bin:$PATH
```

Comment:
```suggestion
FROM python:3.9

RUN python -m pip install --disable-pip-version-check --upgrade virtualenv \
 && python -m virtualenv /home/venv --no-download --no-periodic-update

ENV VIRTUAL_ENV="/home/venv"
ENV PATH="/home/venv/bin:$PATH"
```

Reply:
I have updated the dockerfile to use the python 3.9 image provided by the vscode devcontainer project see: https://github.com/devcontainers/images/tree/main/src/python